### PR TITLE
[CI:DOCS] upload a translation file

### PR DESCRIPTION
**Pull request by of2x**
#### What this PR does / why we need it:

it is a simplified chinese translation editon for the podman_tutorial.md

#### How to verify it

None
#### Which issue(s) this PR fixes:
None

#### Special notes for your reviewer:
None

### CODE REVIEW

**Review by vrothberg — COMMENTED** [review]
Thanks a lot for contributing!

In order to pass CI, can you sign your commit via `git commit --amend -s` and repush?

Note that I am unable to review.

**Comment by rhatdan** [comment]
Thanks @of2x Are their any Chinese readers out their to proof read this?  If not we can just merge once it is signed.

File: docs/tutorials/podman_tutorial_cn.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+                  -e HTTPD_CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/httpd/ \
+                  registry.fedoraproject.org/f29/httpd /usr/bin/run-httpd
+```
+
+
+
+
+因为命令中的`-d`参数表明容器以`detached`模式运行，所以Podman会在容器运行后打印容器的ID。
+
+注意为了访问这个HTTP服务器，我们将使用端口转发。成功运行需要slirp4nextns 的 v0.3.0+版本。
```
Comment (ashley-cui) [inline docs/tutorials/podman_tutorial_cn.md:47]:
```suggestion
注意为了访问这个HTTP服务器，我们将使用端口转发。成功运行需要slirp4netns 的 v0.3.0+版本。
```

File: docs/tutorials/podman_tutorial_cn.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
+> - 译文出自：[掘金翻译计划](https://juejin.cn/translate)
```
Comment (ashley-cui) [inline docs/tutorials/podman_tutorial_cn.md:1]: Is this translation generated from the juejin project? juejin.cn? What is the juejin project?

**Review by ashley-cui — COMMENTED** [review]
Other than a question on juejin and a typo fix, LGTM! Checked all the technical jargon too and it seems correct. Thanks for the translation!

(I don't have professional fluency in Chinese but do have native conversational fluency, so if any community members want to give this another review, please do!)

File: docs/tutorials/podman_tutorial_cn.md (hunk 1 above)
Comment (of2x) [reply docs/tutorials/podman_tutorial_cn.md:47]: I'm sorry for the typo errors.

Juejin is a technical community,which encourages us to translate good documents into Chinese.

**Comment by TomSweeneyRedHat** [comment]
@of2x 非常感谢！  In general, looks good to me, but have asked a couple of our Chinese team members to take a look at it.

File: docs/tutorials/podman_tutorial_cn.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+
+
+
+![PODMAN logo](https://github.com/containers/podman/raw/main/logo/podman-logo-source.svg)
+
+Podman是作为libpod库的一部分提供的程序，可以被用于创建和管理容器。
```
Comment (ypu) [inline docs/tutorials/podman_tutorial_cn.md:7]: This should be better:
Podman是由libpod库提供一个实用的程序，可以被用于创建和管理容器。

File: docs/tutorials/podman_tutorial_cn.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+
+下面的教程会教你如何启动 Podman 并使用 Podman 执行一些基本的命令。
+
+
+如果你正在使用 Mac 或者 Windows ，你应该先查看[Mac和Windows使用说明](https://github.com/containers/podman/blob/master/docs/tutorials/mac_win_client.md)来设置 Podman远程客户端。
+
+
+
+**注意**：这些代码示例旨在作为非root的用户运行，并且在需要root权限的时候使用`sudo`。
```
Comment (ypu) [inline docs/tutorials/podman_tutorial_cn.md:16]: 示例中所有命令皆以非root的用户运行，必要的时候通过`sudo`命令来获取root权限。

File: docs/tutorials/podman_tutorial_cn.md [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+
+
+## 安装Podman
+
+
+
+安装或者构建Podman，请参照[安装说明](https://github.com/containers/podman/blob/master/install.md)。
```
Comment (ypu) [inline docs/tutorials/podman_tutorial_cn.md:23]: ”安装或者编译Podman“
Here the word "building" translate to "编译" should be better.

File: docs/tutorials/podman_tutorial_cn.md [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+
+## 熟悉podman
+
+
+### 运行一个示例容器
+
+
+这个示例容器会运行一个简单的只有主页的httpd服务器。
+
+```bash
```
Comment (ypu) [inline docs/tutorials/podman_tutorial_cn.md:35]: Please keep this as console as the original doc. This should be a grammar of md docs, don't need change it.

File: docs/tutorials/podman_tutorial_cn.md [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+
+Podman的`ps`命令用于列出正在创建和运行的容器。
+
+```console
+podman ps
+```
+
+
+
+
```
Comment (ypu) [inline docs/tutorials/podman_tutorial_cn.md:58]: Seems too many blank lines here. This is also happens in other places. Can you help to clean up it?

File: docs/tutorials/podman_tutorial_cn.md [hunk 8]
```diff
@@ -0,0 +1,238 @@
...
+因为命令中的`-d`参数表明容器以`detached`模式运行，所以Podman会在容器运行后打印容器的ID。
+
+注意为了访问这个HTTP服务器，我们将使用端口转发。成功运行需要slirp4nextns 的 v0.3.0+版本。
+
+
+Podman的`ps`命令用于列出正在创建和运行的容器。
```
Comment (ypu) [inline docs/tutorials/podman_tutorial_cn.md:50]: In the original doc it use *ps*, this should also be a grammar of md docs. Please keep it. Also for other places

File: docs/tutorials/podman_tutorial_cn.md [hunk 9]
```diff
@@ -0,0 +1,238 @@
...
+
+```console
+podman ps
+```
+
+
+
+
+**注意**：如果为`ps`命令添加 `-a`参数，Podman 将展示所有的容器。 
+
```
Comment (ypu) [inline docs/tutorials/podman_tutorial_cn.md:60]: Grammar keep for here too.

File: docs/tutorials/podman_tutorial_cn.md [hunk 10]
```diff
@@ -0,0 +1,238 @@
...
+
+
+### 查看正在运行的容器
+
+
+
+你可以`inspect` (查看)一个正在运行的容器的元数据以及其他详细信息。我们甚至可以使用 `inspect`的子命令查看分配给容器的IP地址。由于容器以非root模式运行，没有分配IP地址，`inspect`的输出会是 none
```
Comment (ypu) [inline docs/tutorials/podman_tutorial_cn.md:67]: The"`" is different from " ' ". Please pay more attention in your translation. I see it also exist in other places. Can you help to go through the doc and fix it?

File: docs/tutorials/podman_tutorial_cn.md [hunk 11]
```diff
@@ -0,0 +1,238 @@
...
+
+
+
+### 迁移容器
+
+
+
+为了将容器从一个主机上热迁移到另一个主机，容器在源系统上创建检查点，传输到目的系统，然后再目的系统上恢复。
+
+当传输检查点时，可以指定一个输出文件。
```
Comment (ypu) [inline docs/tutorials/podman_tutorial_cn.md:165]: 为了便于传输容器的检查点，可以将其存储在一个指定的输出文件中。

File: docs/tutorials/podman_tutorial_cn.md [hunk 12]
```diff
@@ -0,0 +1,238 @@
...
+podman rm --latest
+```
+
+
+你可以使用`podman ps -a` 验证容器的删除。
+
+
+## 集成测试
+
+在环境中如何设置并运行集成测试请查看 集成测试的[自述页面](https://github.com/containers/podman/blob/main/test/README.md)
```
Comment (ypu) [inline docs/tutorials/podman_tutorial_cn.md:228]: For a translate file, maybe it is better to keep the link the same as the original one?

**Comment by ypu** [comment]
@of2x Thanks a lot for the translation. Overall it looks very good. Just left some small comments in the file for some words to make it more suitable for the habit of programmer.

**Comment by TomSweeneyRedHat** [comment]
@ypu TYVM for the review!

File: docs/tutorials/podman_tutorial_cn.md (hunk 7 above)
Comment (ypu) [reply docs/tutorials/podman_tutorial_cn.md:50]: Hello @of2x Still have 5 blank lines here. Can you help to check it?

File: docs/tutorials/podman_tutorial_cn.md (hunk 11 above)
Comment (ypu) [reply docs/tutorials/podman_tutorial_cn.md:154]: Hello @of2x the translation of this sentence seems a little out of context if we just translate it word by word. We can use context and make it more readable in Chinese.

File: docs/tutorials/podman_tutorial_cn.md [hunk 13]
```diff
@@ -0,0 +1,188 @@
+> - 译文出自：[掘金翻译计划](https://juejin.cn/translate)
+
+![PODMAN logo](../../logo/podman-logo-source.svg)
+
+Podman 是作为 libpod 库的一部分提供的程序，可以被用于创建和管理容器。
```
Comment (ypu) [inline docs/tutorials/podman_tutorial_cn.md:5]: Hi @of2x this is not very readable in Chinese. Too many adjective here make the sentence's structure a little complex. Maybe you can separate it to two sentences. Something like:
Podman是由libpod库提供的程序，它是libpod项目的一个组成部分。

**Comment by ypu** [comment]
Hi @of2x left some comments in the file. And have a small advise about the pullreq. As the three commits are fix one problem, it may be better to combine them to one commit, and use git push --force to update it to your branch. 

And as @vrothberg 's comments in the pullreq, you also need setup your git config following:
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address#setting-your-commit-email-address-in-git
https://docs.github.com/en/get-started/getting-started-with-git/setting-your-username-in-git

And generate the commit with:
git commit -s ...

So we can see your email and name in the commit message. And your work is recorded more clearly in the project.

**Comment by ypu** [comment]
> @ypu TYVM for the review!

My pleasure. :)

**Comment by umohnani8** [comment]
@of2x looks like you have some trailing whitespace causing the validate test to fail. Please fix that and we can get this merged

**Comment by rhatdan** [comment]
Probably a little late in life to learn Chinese.
/lgtm
/approve
Thanks @of2x

**Comment by rhatdan** [comment]
@of2x Interested in translating Man pages, or do they move too quickly?

**Comment by TomSweeneyRedHat** [comment]
非常感谢@of2x和@ypu!

**Pull request merged**